### PR TITLE
Chk 2808 edit api response for transaction cart

### PR DESCRIPTION
**Pull request by a-mannia**
#### List of Changes

- Add configuration for run the project in local env.
- Change structure of the apis response by moving the field idTransaction from  PaymentDetailInfoDto to PaymentInfoDto
- Change structure of the apis response by adding the field amount on PaymentDetailInfoDto.

#### Motivation and Context
The goal of the change is to correctly manage the data for a transaction made with a payment advice cart.

#### How Has This Been Tested?
Unit Test

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/test/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PMTransactionDataProviderTest.kt [hunk 1]
```diff
@@ -117,14 +117,15 @@ class PMTransactionDataProviderTest {
...
+                            .idTransaction("MIUR20191119222949")
                             .details(
                                 listOf(
                                     PaymentDetailInfoDto()
                                         .subject(
                                             "/RFB/718173815252003/0.10/TXT/Pagamento di test 20"
                                         )
                                         .iuv("000000044060814")
-                                        .idTransaction("MIUR20191119222949")
+                                        .amount(null)
```
Comment (pietro-tota) [inline src/test/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PMTransactionDataProviderTest.kt:128]:
```suggestion
                                        .amount(500)
```

File: src/test/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PMTransactionDataProviderTest.kt [hunk 2]
```diff
@@ -217,14 +218,15 @@ class PMTransactionDataProviderTest {
...
```
Comment (pietro-tota) [inline src/test/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PMTransactionDataProviderTest.kt:229]:
```suggestion
                                        .amount(500)
```

**Review by pietro-tota — APPROVED** [review]
Well done, just a trivial issue

File: src/test/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PMTransactionDataProviderTest.kt (hunk 1 above)
Comment (a-mannia) [reply src/test/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PMTransactionDataProviderTest.kt]: Done.

File: src/test/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PMTransactionDataProviderTest.kt (hunk 2 above)
Comment (a-mannia) [reply src/test/kotlin/it/pagopa/ecommerce/helpdesk/dataproviders/oracle/PMTransactionDataProviderTest.kt]: Done.

**Review by infantesimone — APPROVED** [review]

**Pull request merged**